scripts/examples/python/mini-xenrt.py

```diff
@@ -120,7 +120,7 @@ def make_operation_list(vm):
             allowed_ops = vms[vm]["allowed_operations"]
             for op in [ "clean_reboot", "suspend", "pool_migrate" ]:
                 if op not in allowed_ops:
-                    raise "VM %s is not in a state where it can %s" % (vms[vm]["name_label"], op)
+                    raise Exception("VM %s is not in a state where it can %s" % (vms[vm]["name_label"], op))
```

Comment:
```suggestion
                    raise RuntimeError("VM %s is not in a state where it can %s" % (vms[vm]["name_label"], op))
```
raise a [RuntimeError](https://docs.python.org/3/library/exceptions.html#RuntimeError)
* Raised when an error is detected that doesn’t fall in any of the other categories. The associated value is a string indicating what precisely went wrong.

Having the information which class of exception 
Reference 1: https://docs.python.org/3/library/exceptions.html

Reference 2: https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/broad-exception-raised.html

> Raising exceptions that are too generic force you to catch exceptions generically too. It will force you to use a naked ``except Exception:`` clause. You might then end up catching exceptions other than the ones you expect to catch. This can hide bugs or make it harder to debug programs when unrelated errors are hidden.

Reply:
This code will be addressed too.

Reply:
Exception has been corrected ,commit id "862b8140d91f56e9a8e9fa4cceb83a319a5ca4d2"